data/uvepss/config.xml

```diff
@@ -0,0 +1,80 @@
+<!-- University of Victory Endings Project Static Site configuration -->
+<!-- See, e.g., https://endings.uvic.ca/staticSearch/docs/howDoIUseIt.html -->
+<config xmlns="http://hcmc.uvic.ca/ns/staticSearch" version="1">
+  <params>
+    <searchFile>../../vol/search.html</searchFile>
+    <recurse>true</recurse>
+    <stopwordsFile>stopwords_en.txt</stopwordsFile>
+    <dictionaryFile>words_en.txt</dictionaryFile>
+    <!-- above required; below optional -->
+    <maxKwicsToHarvest>12</maxKwicsToHarvest> <!-- ignored because phrasalSearch is true -->
+    <maxKwicsToShow>5</maxKwicsToShow>
+    <versionFile>version.txt</versionFile>
+    <createContexts>true</createContexts>
+    <phrasalSearch>true</phrasalSearch>
+    <scoringAlgorithm>raw</scoringAlgorithm> <!-- using "tf-idf" gave long values < 0 which folks did not find very helpful -->
+    <totalKwicLength>11</totalKwicLength>
+    <kwicTruncateString>…</kwicTruncateString>
+    <verbose>false</verbose> 	<!-- comment out for sydb_uvepss_02, but must leave uncommented for 1.4.5;  turn on for debugging index process -->
+    <indentJSON>true</indentJSON> <!-- turn on for debugging -->
+    <outputFolder>uvepss</outputFolder>
+  </params>
+  <rules>
+    <rule match="/html/body//text()[not( ancestor::div[@id eq 'mainContent'] )]" weight="0"/>
+  </rules>
+  <contexts>
+    <context label="headings" match="h1 | h2 | h3 | h4 | h5 | h6"/>
+    <context label="abstracts" match="div[ @id eq 'abstract']"/>
+    <context label="captions" match="div[ @class eq 'caption']"/>
+    <context label="notes" match="div[ @id eq 'notes']"/>
+    <context label="works cited" match="div[ @id eq 'worksCited']"/>
+    <!--
+	List of IANA language tags generated by:
+	1. Generating an XML file of all registerted language tags
+           using data/uvepss/IANA_semantic_to_contexts.xslt.
+	2. Generating a list of all language tags used in DHQ articles
+ 	   i.e., extracting the XPath
+	   //@xml:lang[ normalize-space(.) ne '']!distinct-values()
+	   (untested).
+	3. For each @xml:lang, if it is in the registry file return
+	   the description or descriptions; if not return null.
+	4. Take the descriptions and insert them into the @label
+	   attributes and the languages into the @match attributes
+	   of the <context>.
+	5. Inset the <context>s so generated into this <contexts>.
+        The following does steps 2–4 at one shot on Syd’s machine,
+	presuming the registry is in its default location,
+	/tmp/IANA_language_subtag_registry_semantic.xml:
+	$ for l in $(xsel -t -m "//@xml:lang[not(normalize-space(.)='')]" -v "." -n articles/0*/0*.xml 2>/dev/null | sort | uniq); do echo "=========$l:" ; xsel -t -m "/*/language[@subtag='$l']" -o '    <context label="language: ' -v "description[1]" -i "description[2]" -o " or " -v "description[2]" -b -o "&quot; match=&quot;text()[ lang('$l') ]&quot;/>" -n /tmp/IANA_language_subtag_registry_semantic.xml ; done | perl -pe 's,&lt;,<,g; s,&gt;,>,g; s,&amp;quot;,",g;'
+    -->
```

Comment:
Yes and no. The procedure could well be made a lot simpler, likely even 1 step, but I have (so far) been unable to find a directly web-accessible version of the IANA language registry in either XML or JSON. So it will have to read the plain text, parse it, etc. Not that particularly difficult to do, but since I have routines that read in the JSON version and write out XML already written, since neither the IANA registry nor the set of languages in DHQ changes that fast, and (with the help of that comment) I can do the entire process in just under 4 minutes, I was not making it a high priority.
(That said, it might be a fun exercise to do over break. Maybe when I need a break from dates. :-)